tdrs-backend/reports/zap.conf

```diff
@@ -1,2 +1,120 @@
-10036	IGNORE	(Server leaks version information via Server HTTP Response Header field)
-40025	IGNORE	(Proxy Disclosure)
+# zap-full-scan rule configuration file for tdrs-backend
```

Comment:
ZAP expects all rules to be declared in the config file, and defaults all others to `WARN`. Since we don't trigger failures on WARN level alerts we must set all rules we want to fail on explicitly to `FAIL`.